src/components/blocks/CampusWeatherDetail/CampusWeatherCurrentConditions/CampusWeatherCurrentConditions.tsx

```diff
@@ -0,0 +1,21 @@
+import { celsiusToFahrenheit, getCompassDirection, kphToMph } from '@/util/campusUnitConversion'
+import styles from './CampusWeatherCurrentConditions.module.scss'
+import { CampusWeatherCurrentValue } from './CampusWeatherCurrentValue/CampusWeatherCurrentValue'
+
+export const CampusWeatherCurrentConditions = ({ currentWeatherData }) => {
```

Comment:
Lets discuss a better way to name these

Reply:
commit ID: f7dfa6ef599fddbd67808c4341a50b14aae11530

addressed